app/src/componentLibrary/attributeEditor/attributeEditor.jsx

```diff
@@ -134,6 +134,16 @@ export const AttributeEditor = ({
   };
   const handleAttributeKeyInputChange = (text) => setState({ ...state, isKeyEdited: !!text });
 
+  const isValidForm = !isFormValid();
```

Comment:
Seems like your variable contains the `is form invalid` state as `!isFormValid()` is opposite to the `is form valid`.

Reply:
Oh, yes! You are absolutely right.